biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -313,28 +329,9 @@ real(r8) :: tcwood             ! Temperature response function for wood
                                            gb_mol,                   & ! out
                                            ceair)                      ! out
 
-               ! Part V.  Pre-process some variables that are PFT dependent
-               ! but not environmentally dependent
-               ! ------------------------------------------------------------------------
-
-               do ft = 1,numpft
-
-                  
-                  
-
-                  ! This is probably unnecessary and already calculated
-                  ! ALSO, THIS ROOTING PROFILE IS USED TO CALCULATE RESPIRATION
-                  ! YET IT USES THE PROFILE THAT IS CONSISTENT WITH WATER UPTAKE
-                  ! AND NOT THE PROFILE WE USE FOR DECOMPOSITION
-                  ! SEEMS LIKE THE LATTER WOULD BE MORE APPROPRIATE, RIGHT? (RGK 05-2018)
-                  call set_root_fraction(currentPatch%rootfr_ft(ft,1:bc_in(s)%nlevsoil), ft, &
-                       bc_in(s)%zi_sisl,lowerb=lbound(bc_in(s)%zi_sisl,1), &
-                       icontext = i_hydro_rootprof_context)
-                  
-               end do !ft 
-
                
 
+
                ! ------------------------------------------------------------------------
                ! Part VI: Loop over all leaf layers.
```

Comment:
Minor correction: renumber parts [given move](https://github.com/rgknox/fates/commit/3234d4ad23d54a577874e98e06d8420362210fe0#diff-758dbf12610469ab3e6dc7ca96e76e27) of Part V: `set_root_fraction` to above the do loop.

Reply:
nice catch, will do